public/hide.js

```diff
@@ -3,4 +3,6 @@ if (!document.querySelector('input.filter__box:checked')) {
   document.getElementById('searchresults').className = 'row searchresults';
   document.getElementById('filtercolumn').className = 'filtercolumn';
   document.getElementById('control--filters').className = 'control control--filters';
+  document.getElementById('fb').className = 'control__button';
```

Comment:
Does this code only get run on a specific route?
(_if not it's dupe of `client/routes/search.js` ..._)